core/templates/core/base.html

```diff
@@ -13,29 +13,34 @@
         <script src="https://cdn.tailwindcss.com"></script>
         {% load static %}
         <script src="{% static "tailwind_custom_styles.js" %}"></script>
-        <script src="https://unpkg.com/starback@2.1.1/dist/starback.global.js"></script>
 
         <link href="{% static 'fontawesomefree/css/fontawesome.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/brands.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/solid.css' %}" rel="stylesheet" type="text/css">
         {% block head %}
         {% endblock %}
     </head>
-    <body class="bg-black text-white">
-        <main class="w-screen h-screen">
-            <nav class="flex flex-wrap justify-between items-center w-screen p-2 bg-black border-b-2 border-white">
-                <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center text-gray-300 transition-colors duration-300 ml-5 hover:text-blue-100" target="_blank">
-                    <i class="fa-brands fa-github fa-3x mr-5"></i>
-                    <span class="text-2xl text-gray-300 font-bold my-4">GwardiaHub</span>
-                    <ul class="flex">
+    <body class="bg-gray-900 text-gray-50">
+        <main class="flex h-screen w-screen max-w-[100vw] max-h-[100vh] flex-col">
+            <nav class="flex flex-nowrap justify-between items-center w-full p-1 bg-gray-950 border-b border-gray-600">
+                <div class="flex flex-nowrap">
+                    <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center text-gray-300 transition-colors duration-300 ml-2 md:ml-5 hover:text-blue-100" target="_blank">
+                        <i class="fa-brands fa-github fa-2x mr-5"></i>
+                    </a>
+                    <a href="/" class="text-md md:text-xl text-gray-300 font-bold my-3">GwardiaHub</a>
+                </div>
+                <div class="m-2">
+                    <ul class="flex justify-between items-center">
+                        {% block navbar_links %}
+                        {% endblock %}
                         <li>
-                            <a href="{% url 'core:index' %}" class="mr-5 text-gray-300 hover:underline" aria-current="page">Strona główna</a>
-                        </li>
-                        <li>
-                            <a href="{% url 'gwardia:index' %}" class="mr-5 text-gray-300 hover:underline" aria-current="page">Gwardia</a>
+                            <a type="button" class="inline-block" href="https://link.szczurek.yt/hmmm">
```

Comment:
Make it button, not `a`. And it is too obvious, figure out something smarter to rickroll people